internal/bulk.go

```diff
@@ -69,15 +73,15 @@ func Bulk(ctx context.Context, in io.Reader) <-chan *BulkResponse {
 			}
 			wg.Add(1)
 			go func() {
-				resCh <- processRequest(ctx, req, seq)
+				resCh <- processRequest(ctx, opts, req, seq)
 				wg.Done()
 			}()
 		}
 	}()
 	return resCh
 }
 
-func processRequest(ctx context.Context, req BulkRequest, seq int64) *BulkResponse {
+func processRequest(ctx context.Context, b *BulkOptions, req BulkRequest, seq int64) *BulkResponse {
```

Comment:
Probably best with `opts` at end.

Reply:
I called it `b` because the function is already creating an `opts` variable inside (and I saw somewhere in the code an `{{Something}}Options` variable named with the first letter of "something". But I'll give it a second thought including the other changes we discussed above.

Reply:
Amended in my last push